Tests/GRPCCoreTests/Call/Client/Internal/ClientRPCExecutorTests.swift

```diff
@@ -223,4 +225,47 @@ final class ClientRPCExecutorTests: XCTestCase {
     XCTAssertEqual(tester.clientStreamOpenFailures, 1)
     XCTAssertEqual(tester.serverStreamsAccepted, 0)
   }
+
+  func testTimeoutIsPropagated() async throws {
+    var policies: [RPCExecutionPolicy?] = [nil]
```

Comment:
This `nil` tests the one-shot executor, right? Maybe it's a bit annoying but do you mind adding a comment explaining this? It's not immediately obvious from looking at the test.